build/concat.js

```diff
@@ -1,14 +1,34 @@
-import gulp from 'gulp';
-import gulpConcat from 'gulp-concat';
 import paths from '../mconfig.json';
+import message from './utils/message.js';
+import fs from 'fs';
+import concat from 'concat';
 
-function concat() {
-    return gulp
-        .src([
-            `${paths.scripts.vendors.src}*.js`
-        ])
-        .pipe(gulpConcat(`${paths.scripts.vendors.main}.js`))
-        .pipe(gulp.dest(paths.scripts.dest));
-}
+export function concatVendors() {
+    console.time('Concat in');
 
-export default concat;
+    // Get all files in scripts/vendors/
+    const files = fs.readdirSync(paths.scripts.vendors.src);
+
+    // Extract no-JS files
+    var jsFiles = files.filter((f)=>{
+      if(f.includes('.js')) {
+        return true;
+      } else {
+        return false;
+      }
+    });
+
+    // Add absolute path
+    jsFiles = jsFiles.map((f) => {
+      return `${paths.scripts.vendors.src + f}`;
+    });
```

Comment:
```suggestion
    // Exclude files that are not JavaScript
    var jsFiles = files.filter((file) => {
        return file.includes('.js');
    });

    // Prepend absolute path
    jsFiles = jsFiles.map((file) => {
        return `${paths.scripts.vendors.src + file}`;
    });
```

The `String.prototype.includes()` method returns a boolean, no need for `if else`.